pom.xml

```diff
@@ -147,7 +147,7 @@
         <scala.version>2.11.8</scala.version>
         <semver4j.version>2.2.0-graylog.1</semver4j.version>
         <shiro.version>1.5.2</shiro.version>
-        <sigar.version>1.6.4</sigar.version>
+        <oshi.version>5.3.6</oshi.version>
```

Comment:
Can you please update to 5.3.7 release of oshi?

Reply:
can do. but when I did this PR it wasn't even released :man_shrugging: